lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -472,7 +483,10 @@ def buildUI(self):
                 if schemaTypeName == 'UsdGeomXformable':
                     self.createTransformAttributesSection(sectionName, attrsToAdd)
                 else:
-                    self.createSection(sectionName, attrsToAdd)
+                    sectionsToCollapse = ['Curves', 'Point Based', 'Geometric Prim', 'Boundable',
+                                          'Imageable', 'Field Asset', 'Light']
+                    collapse = sectionName in sectionsToCollapse
+                    self.createSection(sectionName, attrsToAdd, collapse)
```

Comment:
This is a simple fix for another JIRA item (to collapse certain sections by default).

Reply:
I guess the hard-coded list is O.K. because it won't change much?

Reply:
Right it was a list given to me in the JIRA item. It would only change if Nat wants a change.